src/common/snippets/src/lowered/pass/insert_buffers.cpp

```diff
@@ -28,6 +29,39 @@ std::vector<size_t> get_buffer_loop_ids(const std::vector<size_t>& lhs, const st
     }
     return buffer_loop_ids;
 }
+
+ov::Shape compute_allocation_shape(const LinearIR::LoopManagerPtr& loop_manager,
```

Comment:
As I understand it's temporary solution. Could you please create the ticket for that and attach to the Buffer refactoring ticket 113664? I have some thoughts about this problem how we can implement it, probably. I'd like to write them to the ticket comment. Because at the moment the solution works but should be upgraded anyway
Thanks!

Reply:
The ticket has been created and linked to the 113664. Thanks!